src/app/grid/grid-validator-service-extended/grid-validator-service-extended.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="grid__wrapper">
+    <igx-grid #grid1 [data]="data" [width]="'100%'" [height]="'480px'" [autoGenerate]="false" [batchEditing]="true"
+        [primaryKey]="'id'" (onFormGroupCreate)='formCreateHandler($event)'>
+        <igx-column field="Avatar" header="Photo" dataType="string" width="80" [editable]="false">
+            <ng-template igxCell let-cell="cell">
+                <div class="cell__inner avatar-cell">
+                    <igx-avatar [src]="cell.row.data.avatar" [roundShape]="true" size="small"></igx-avatar>
+                </div>
+            </ng-template>
+        </igx-column>
+        <igx-column field="name" forbiddenName="josh" header="Name" [editable]="true" required>
+            <ng-template igxCellValidationError let-cell='cell'>
+                <ng-container *ngTemplateOutlet="cell.defaultErrorTemplate">
+                </ng-container>
+                <div *ngIf="cell.formGroup?.get(cell.column?.field).errors?.['forbiddenName']">
+                    This name is forbidden.
+                </div>
+            </ng-template>
+        </igx-column>
+        <igx-column field="company" header="Company" [editable]="true"></igx-column>
+        <igx-column field="email" width="190" header="Email" [editable]="true" required email></igx-column>
+        <igx-column field="fax" header="Phone" [editable]="true">
+            <ng-template igxCellValidationError let-cell='cell'>
+                <ng-container *ngTemplateOutlet="cell.defaultErrorTemplate">
+                </ng-container>
+                <div *ngIf="cell.formGroup?.get(cell.column?.field).errors?.['forbiddenFax']">
```

Comment:
Custom error temlplate api changes a bit. Should be updated as per the spec: https://github.com/IgniteUI/igniteui-angular/wiki/Grid-Validation-Specification